tequilapi/endpoints/auth.go

```diff
@@ -142,13 +149,81 @@ func (api *authenticationAPI) Login(c *gin.Context) {
 	utils.WriteAsJSON(response, c.Writer)
 }
 
+// swagger:operation GET /auth/login-mystnodes SSO LoginMystnodesInit
+// ---
+// summary: LoginMystnodesInit
+// description: SSO init endpoint to auth via mystnodes
+//
+// responses:
+//
+//	200:
+//	  description: link response
+//	  schema:
+//	    "$ref": "#/definitions/MystnodesSSOLinkResponse"
+func (api *authenticationAPI) LoginMystnodesInit(c *gin.Context) {
+	link, err := api.ssoMystnodes.SSOLink(c.Request.Header.Get("X-Forwarded-Host"))
```

Comment:
I am not sure about this.

I take this header which is set by tequilapi reverse proxy to be used for **redirectUrl** generation